ui/src/views/mint/mint.context.tsx

```diff
@@ -114,8 +119,15 @@ export abstract class Mint {
           config={{
             transaction: {
               onSuccess: (data) => {
-                console.log('Successfully minted! what now?', data);
-                alert('transaction hash: ' + data.transactionHash);
+                // AppLog.info('Transaction hash: ' + data);
+                pushToast('success', 'Successfully minted!', {
+                  onDismiss: () => setMintSuccess(true),
+                  duration: 2000,
+                });

```

Comment:
I think you don't need the toast dismiss to show up the success string, both may be triggered at the same time so user do not need to wait these 2 seconds to interact.